src/pages/signup/Signup.tsx

```diff
@@ -171,6 +175,17 @@ const Signup: React.FC = () => {
     }
   };
 
+  // sends a verication link to the user's email //
+  const emailVerification = () => {
+    const user = getAuth();
```

Comment:
Can we use auth instead of getAuth? Because it might not work on iOS and Android.

Reply:
auth variable from firebase.js

Reply:
changed it to using auth instead of getauth!